client/src/app/site/site.component.html

```diff
@@ -1,10 +1,12 @@
-<os-banner></os-banner>
+<os-banner [osRepoObservable]="pollRepo"></os-banner>
```

Comment:
Do not use this directive for this usecase and remove it. If you need always-present data look in the active-meeting service. At the bottom data for the active meeting is requested. Add your data there with a comment why it is needed. (E.g. I added the projectors there since they are always everywhere needed).